Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -46,7 +46,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Required. Initialize session. automaticallyDisplayDeepLinkController is optional (default is NO).
     [branch initSessionWithLaunchOptions:launchOptions automaticallyDisplayDeepLinkController:YES deepLinkHandler:^(NSDictionary *params, NSError *error) {
         if (!error) {
+
             NSLog(@"initSession succeeded with params: %@", params);
+            NSDictionary *searchAds = [BNCPreferenceHelper preferenceHelper].appleSearchAdDetails;
+            NSLog(@"Search Ads:\n%@.", searchAds);    //  Apple Search ad demo.
+            NSLog(@"");
```

Comment:
No.  But it's in the test bed app (and not the SDK) so it'll be easier for Deepa to test the code.  

I'll take it out next release cycle.